deployment.toml

```diff
@@ -1,27 +1,32 @@
-# [[cells]]
-# name = "my_cell"
-# enable_type_id = false
-# location = { file = "build/release/my_cell" }
-#
-# # reference to on-chain cells
-# [[cells]]
-# name = "genesis_cell"
-# enable_type_id = false
-# location = { tx_hash = "0x71a7ba8fc96349fea0ed3a5c47992e3b4084b031a42264a018e0072e8172e46c", index = 0 }
- 
+[[cells]]
+name = "ckb-cheque-script"
+enable_type_id = true
+location = { file = "build/release/ckb-cheque-script" }
 
-# # Dep group cells
-# [[dep_groups]]
-# name = "my_dep_group"
-# cells = [
-#   "my_cell",
-#   "genesis_cell"
-# ]
+# reference to on-chain cells
+[[cells]]
+name = "secp256k1_data"
+enable_type_id = false
+location = { tx_hash = "0xf8de3bb47d055cdf460d93a2a6e1b05f7432f9777c8c474abf4eec1d4aee5d37", index = 3 } 
```

Comment:
Do users also need to modify this? If the user needs to modify, we can also consider adding a comment.

Reply:
OK. I will give a comment here.